server/webapp/WEB-INF/rails.new/app/presenters/api_v1/dashboard/pipeline_pause_info_representer.rb

```diff
@@ -0,0 +1,31 @@
+##########################################################################
+# Copyright 2015 ThoughtWorks, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##########################################################################
+
+module ApiV1
+  module Dashboard
+    class PipelinePauseInfoRepresenter < ApiV1::BaseRepresenter
+      alias_method :pipeline_pause_info, :represented
+
+      property :paused, as: :paused
+      property :pauseBy, as: :paused_by, getter: lambda { |options|
+                         pauseBy.empty? ? nil : pauseBy }
+      property :pauseCause, as: :pause_reason, getter: lambda { |options|
+                            pauseCause.empty? ? nil : pauseCause }
```

Comment:
Use `blank?` instead of `empty?` — see http://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails